extensions/resource-deployment/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "resource-deployment",
+  "displayName": "Azure Data Resource Deployment",
+  "description": "Azure Data Resource Deployment",
+  "version": "0.0.1",
+  "publisher": "Microsoft",
+  "preview": true,
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
+  "icon": "images/sqlserver.png",
+  "aiKey": "AIF-5574968e-856d-40d2-af67-c89a14e76412",
+  "engines": {
+    "vscode": "*",
+    "azdata": "^1.6.0"
+  },
+  "activationEvents": [
+    "onCommand:azdata.resource.sql-image.deploy",
+    "onCommand:azdata.resource.sql-bdc.deploy"
+  ],
+  "main": "./out/main",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/azuredatastudio.git"
+  },
+  "extensionDependencies": [
+    "Microsoft.mssql"
+  ],
+  "contributes": {
+    "commands": [
+      {
+        "command": "azdata.resource.sql-image.deploy",
+        "title": "%deploy-sql-image-command-name%",
+        "category": "%deploy-resource-command-category%"
+      },
+      {
+        "command": "azdata.resource.sql-bdc.deploy",
+        "title": "%deploy-sql-bdc-command-name%",
+        "category": "%deploy-resource-command-category%"
+      }
+    ],
+    "dataExplorer.actions": [
+      {
+        "commandId": "azdata.resource.sql-image.deploy",
+        "label": "%deploy-sql-image-menu-item-text%",
+        "category": "%deploy-resource-command-category%",
+        "isPrimary": false
+      },
+      {
+        "commandId": "azdata.resource.sql-bdc.deploy",
+        "label": "%deploy-sql-bdc-menu-item-text%",
+        "category": "%deploy-resource-command-category%",
+        "isPrimary": false
+      }
+    ]
+  },
+  "dependencies": {
+    "vscode-nls": "^3.2.1",
+    "download": "^6.2.5",
+    "shelljs": "^0.8.3"
```

Comment:
Neither of these appear to be used

Reply:
will be used when we are actually implementing the features

Reply:
I would still suggest removing them until they're actually needed. That way if they end up not actually being necessary then we aren't needlessly inflating the size of the extension.

Reply:
sure, vscode-nls is still needed for localization.